dataland-frontend/src/components/forms/parts/fields/StringBaseDataPointFormField.vue

```diff
@@ -1,16 +1,20 @@
 <template>
   <div class="form-field">
-    <FormKit v-model="baseDataPoint" type="group" name="dataSource">
+    <FormKit v-model="baseDataPoint" type="group" ignore="true">
```

Comment:
Does the ignore flag work this way? Mustn't it be `:ignore="true"` or just `ignore`?

Reply:
Must this even be a formkit now? There is no FormkitField inside.

Reply:
This v-model seems unused to me as well.

Reply:
Not my code.
Git says @pioSko2

Reply:
it is being used. Both this input and the hidden dataSource inputs use v-model in order to update each other. When the user uses the visible inputs, they update the hidden ones. When the user opens the Edit page, the hidden inputs are first populated (from the incoming data) and they in turn update the visible inputs visible to the user.

Reply:
Ahh yeah :) Done.

Reply:
Why not remove the whole formkit tag?

Reply:
hrmm.. wow didn't notice. Done.